container_pipeline/workers/build.py

```diff
@@ -35,11 +35,11 @@ def handle_job(self, job):
                 is_build_running
 
         if parent_build_running:
-            self.logger.info('Parents in build: {}, pushing job: {} back '
+            self.logger.debug('Parents in build: {}, pushing job: {} back '
```

Comment:
I think this message should stay as `info`. This message has information and reason for pushing the job back to queue. And I think it is a clue for log viewer to understand why job is pushed back to queue.